matsim/src/main/java/org/matsim/analysis/CalcLinkStats.java

```diff
@@ -70,7 +70,7 @@ public LinkData(final double[][] linksVolumes, final double[][] linksTTimes) {
 	@Inject
 	public CalcLinkStats(final Network network) {
 		this.network = network;
-		this.linkData = new TreeMap<>();
+		this.linkData = new ConcurrentHashMap<>();
```

Comment:
I can see that `linkData ` is modified only in `public void reset()`, not sure why it is `ConcurrentHashMap`